back/src/domain/offer/entities/EstablishmentLeadEntity.ts

```diff
@@ -14,12 +14,24 @@ const establishmentLeadEventKind = [
 
 type EstablishmentLeadEventKind = (typeof establishmentLeadEventKind)[number];
 
-type EstablishmentLeadEvent = {
-  occuredAt: Date;
-  kind: EstablishmentLeadEventKind;
-  conventionId: ConventionId;
-  notification?: { id: NotificationId; kind: NotificationKind };
-};
+export type EstablishmentLeadEvent =
+  | {
+      kind: Extract<EstablishmentLeadEventKind, "to-be-reminded">;
```

Comment:
c'est mieux d'utiliser l'utilitaire : `ExtractFromExisting`, car ca évite les fautes d'orthographe. En l'état actuel, tu pourrais écrire `Extract<EstablishmentLeadEventKind, "to-be-reminded-wrong">;` sans problème de compilation.

```suggestion
      kind: ExtractFromExisting<EstablishmentLeadEventKind, "to-be-reminded">;
```